TikTacToeGame/app/src/main/java/com/r4ziel/tiktactoegame/modules/ViewModelModule.kt

```diff
@@ -8,5 +8,5 @@ import org.koin.dsl.module
  */
 
 val viewModelModule = module {
-    single { GameFragmentViewModel() }
+    single { GameFragmentViewModel(get()) }
```

Comment:
FYI: Koin has a new feature where you no longer have to add a get() for each constructor parameter. 

```suggestion
    singleOf(::GameFragmentViewModel)
```